lib/notifications/display.dart

```diff
@@ -261,9 +261,10 @@ class NotificationDisplayManager {
     // the first.
     messagingStyle.conversationTitle = switch (data.recipient) {
       FcmMessageChannelRecipient(:var streamName?, :var topic) =>
-        '#$streamName > ${topic.displayName}',
+        zulipLocalizations.notifChannelConversationLabel(streamName, topic.displayName),
```

Comment:
I don't think translating this makes sense: the "#" and ">" come from how Zulip displays channels and topics, not from any convention of English punctuation. We use the "#" in icons, and we use both the "#" and ">" in the Markdown syntax.

Because they don't come from English punctuation conventions, it doesn't make sense to try to find their equivalents in other languages. And because they do come from how Zulip displays channels and topics in other contexts, if they got translated here to anything else it would just cause confusion.

Reply:
My thought was:

https://github.com/zulip/zulip-flutter/pull/1148#discussion_r1915755966
> Huh, yeah I guess maybe this could look different in RTL languages, or languages that use a space character other than " "?

but

- I'm not confident about the RTL point; I'd want feedback from a regular user of RTL interfaces
- The space-character thing is a nit, especially since (AIUI) this string doesn't appear in a paragraph (where localized space characters would be used)

and indeed, incorrect/inconsistent translations is a possibility; helpful to prevent when possible.